apps/pie-docs/src/foundations/iconography/code.md

```diff
@@ -0,0 +1,277 @@
+
+---
+
+## Pie Icons
+
+`@justeattakeaway/pie-icons` is our base icon package. It is a collection of SVG files. This means that you can use these icons in all the same ways you can use SVGs (e.g. img, background-image, inline, object, embed, iframe).
+
+[This package](https://github.com/justeattakeaway/pie/tree/main/packages/tools/pie-icons) is used to build the framework-specific icons for Web Components, React and Vue.
+
+
+{% notification {
+  type: "warning",
+  message: "Whilst this package can be used for web applications, we strongly advise that you use a framework-specific implementation instead."
+} %}
+
+### Usage - Client-side Javascript
+
+To use the icons in your application, you need to include the package in your project, add the icons using data-attributes and invoke the `replace()` function:
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons
+```
+
+#### Include the package
+You can load directly from the installed package, or from a CDN provider.
+
+```html
+<script src="path/to/dist/pie-icons.js"></script>
+```
+```html
+<!-- choose one -->
+<script src="https://unpkg.com/@justeattakeaway/pie-icons"></script>
+<script src="https://cdn.jsdelivr.net/npm/@justeattakeaway/pie-icons/dist/pie-icons.min.js"></script>
+```
+
+#### HTML
+Simply add a `data-pie-icons` attribute with the icon name to an element. You can see all of our icons on the [library page](http://localhost:8080/foundations/iconography/library/).
+
+```html
+<i data-pie-icons="menu"></i>
+```
+
+#### Invoke the package
+Calling the `replace()` function will replace all elements with the `data-pie-icons` attribute with the relevant icon.
+
+```html
+<script>
+  window['pie-icons'].default.replace();
+</script>
+```
+---
+
+### Usage - NodeJS
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons
+```
+
+#### Require the package
+Simply require the package in your application where needed:
+
+```js
+const pieicons = require('@justeattakeaway/pie-icons');
+```
+
+#### Call the APIs
+The package exposes a number of APIs for you to use:
+
+```js
+pieicons.icons.x
+// {
+//    name: 'x',
+//    contents: '<line ... /><line ... />`,
+//    tags: ['cancel', 'close', 'delete', 'remove'],
+//    attrs: {
+//      class: 'c-pieIcon c-pieIcon--x',
+//      xmlns: 'http://www.w3.org/2000/svg',
+//    },
+//    toSvg: [Function],
+// }
+
+pieicons.icons.x.toSvg()
+// <svg class="c-pieIcon c-pieIcon--x" ...><line ... /><line ... /></svg>
+
+pieicons.icons.x.toSvg({ class: 'foo bar', 'stroke-width': 1, color: 'red' })
+// <svg class="c-pieIcon c-pieIcon--x foo bar" stroke-width="1" color="red" ...><line ... /><line ... /></svg>
```

Comment:
I haven't tested this, but I think you currently need to reference `default` in this call (as that looks like what is done in the build scripts atm).

So in the `pie-icons-webc` package we do:

```js
import pieIcons from '@justeattakeaway/pie-icons';
const { icons } = pieIcons.default;

icons.x.toSvg({ class: 'foo bar', 'stroke-width': 1, color: 'red' })
```

Reply:
Think this is still an issue.